Client/Application/ClientPreferences.swift

```diff
@@ -98,8 +98,6 @@ extension Preferences {
         static let backgroundImages = Option<Bool>(key: "newtabpage.background-images", default: true)
         /// Whether sponsored images are included into the background image rotation
         static let backgroundSponsoredImages = Option<Bool>(key: "newtabpage.background-sponsored-images", default: true)
-        /// Whether the iOS keyboard auto-opens on a NTP or not
-        static let autoOpenKeyboard = Option<Bool>(key: "newtabpage.auto-open-keyboard", default: false)
```

Comment:
Yeah, partly because of Brave Today, party because tapping on URL bar will now display an entirely separate VC from NTP with just favs (no more stats, etc.) so